pxr/imaging/hgiMetal/hgi.mm

```diff
@@ -145,11 +146,10 @@
 bool
 HgiMetal::IsBackendSupported() const
 {
-    // Want Metal 2.0 and Metal Shading Language 2.2 or higher.
-    if (@available(macOS 10.15, ios 13.0, *)) {
+    if (@available(ios 14.0, *))
```

Comment:
is no macOS check required any more? aka, will this conditional return true for macos due to the asterisk?

Reply:
I discussed this further on our side - this was an opinion from me as 10.15 is quite old, but this can probably still stay for safety sakes